.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "[python]": {
+    "editor.defaultFormatter": "ms-python.black-formatter"
+  },
+  "python.formatting.provider": "none",
+  "editor.fontLigatures": false,
+  "workbench.colorCustomizations": {},
+  "autopep8.interpreter": ["D:\\Python\\Python3.10\\python.exe"],
```

Comment:
This explicit path is not good, for other people (on other OS) this will be different. Please remove it and set this path as a user setting for you.

Reply:
I've removed the folder and updated .gitignore. It shouldn;t be in git, but just local for me